controls/views.py

```diff
@@ -817,26 +822,33 @@ def restore_to_history(request, smt_id, history_id):
     """
     Restore the current model instance to a previous version
     """
+    full_smt_history = None
+    try:
+        smt = Statement.objects.get(id=smt_id)
+        recent_smt = smt.history.first()
+    except:
```

Comment:
Just in case multiple exceptions get thrown from lower levels, it's usually helpful to specify the particular exception you're trying to handle (and I think it's something Django Doctor likes to see). When using a "get()", I usually do:
`except Statement.DoesNotExist:`

Reply:
Statement.DoesNotExist keep getting to the yellow error page so I optted for ObjectDoesNotExist from django.core.exceptions